### PR TITLE
Create HUNT.py

### PR DESCRIPTION
**Pull request by sbzo**
This only script integrates all previous scripts in one, optimizing time and processing. Also, exclude any analysis from fuzzer.

### CODE REVIEW

File: passive/HUNT.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import re
+from org.zaproxy.zap.extension.script import ScriptVars
+
+''' find posible vulnearble entry points using Hunt Methodology - https://github.com/bugcrowd/HUNT'''
```
[the remark this answers is not in the document]
Comment (sbzo) [reply passive/HUNT.py]: Thanks for review, I made the changes.

File: passive/HUNT.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        return True
+    else:
+        return False
+
+def scan(ps, msg, src):
+    # Test the request and/or response here
+    if ScriptVars.getGlobalVar("hunt") is None:
+        ScriptVars.setGlobalVar("hunt","init")
+
+    if (msg):
```
[the remark this answers is not in the document]
Comment (sbzo) [reply passive/HUNT.py]: I did the change. Thank you.

File: passive/HUNT.py [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        # Change to a test which detects the vulnerability
```
[the remark this answers is not in the document]
Comment (sbzo) [reply passive/HUNT.py]: Removed the comments in this section.

File: passive/HUNT.py [hunk 4]
```diff
@@ -0,0 +1,130 @@
+import re
+from org.zaproxy.zap.extension.script import ScriptVars
+
+''' find posible vulnearble entry points using Hunt Methodology - https://github.com/bugcrowd/HUNT'''
+''' HUNT Scanner for ZAP Proxy (Alpha - Contributed by Ricardo Lobo @_sbzo)'''
+
+def appliesToHistoryType(histType):
+    if (histType<=2):
```
[the remark this answers is not in the document]
Comment (sbzo) [reply passive/HUNT.py]: I left only the reference to the bugcrowd project.

File: passive/HUNT.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (sbzo) [reply passive/HUNT.py]: This point is the same above, I remove the conditional.

**Comment by thc202** [comment]
Thanks!

**Pull request merged**